src/utils/node-features.ts

```diff
@@ -53,3 +53,10 @@ export const importAttributes: Version[] = [
 export const testRunnerGlob: Version[] = [
 	[21, 0, 0],
 ];
+
+// https://nodejs.org/docs/latest-v20.x/api/cli.html#--enable-source-maps
+// https://github.com/nodejs/node/pull/50827
+// Allow using --enable-source-maps together with custom Error.prepareStackTrace
+export const prepareStackTraceWithSourcemap: Version[] = [
+	[20, 12, 0],
```

Comment:
Also `21.6.0`

Seems https://github.com/nodejs/node/pull/50827 is included in the following releases (ordered by release date):
`v21.6.0` https://github.com/nodejs/node/commit/291265ce27
`v21.6.1` https://github.com/nodejs/node/commit/291265ce27
`v21.6.2` https://github.com/nodejs/node/commit/291265ce27
`v21.7.0` https://github.com/nodejs/node/commit/291265ce27
`v21.7.1` https://github.com/nodejs/node/commit/291265ce27
`v20.12.0` https://github.com/nodejs/node/commit/f4d7f0498e
`v21.7.2` https://github.com/nodejs/node/commit/291265ce27
`v20.12.1` https://github.com/nodejs/node/commit/f4d7f0498e